### DISCUSSION TITLE
Modern Material You widgets

### DISCUSSION THREAD

**Discussion by ronniedroid** [FossifyOrg/General-Discussion#636] [Ideas]
I want to add some modern widgets to the clock app.

Should/can I do that?

Should I add new ones or update the old ones?

If I add new ones, can I use Glance? Or should I stick to xml?

**Comment by Aga-C** [comment]
Applying Material You colors to widgets was already requested: https://github.com/FossifyOrg/General-Discussion/issues/146. 

If you really like to work on Clock widgets, there are also two existing issues without `needs triage`, so you can just do them:
- https://github.com/FossifyOrg/Clock/issues/39
- https://github.com/FossifyOrg/Clock/issues/15

> If I add new ones, can I use Glance? Or should I stick to xml?

https://github.com/FossifyOrg/General-Discussion/issues/63#issuecomment-1874371345
